packages/ida.plugin.lighthouse.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,50 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    # Download the plugin
+    $toolName    = "lighthouse"
+    $toolVersion = "0.9.3"
+    $toolDir     = Join-Path ${Env:chocolateyPackageFolder} "temp_$([guid]::NewGuid())"
+    $toolUrl     = "https://github.com/gaasedelen/lighthouse/archive/refs/tags/v0.9.3.zip"
+    $toolSha256  = "b9f1888827f68d69672d1202783b47f7603fb80f42f9b70edeb9349d40c4f33c"
+    $packageArgs =
+    @{
+        packageName   = ${Env:ChocolateyPackageName}
+        unzipLocation = $toolDir
```

Comment:
Use `$pluginsDir` directly as unzip location instead of downloading to `%TEMP%` and copying items over